content/docs/guides/auth/ALTS_GCP.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I think this content would be more powerful as a blog post with more of a walkthrough.